jerry-ext/extfunc/extfunc-print.c

```diff
@@ -0,0 +1,112 @@
+/* Copyright JS Foundation and other contributors, http://js.foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "jerryscript-ext/extfunc.h"
```

Comment:
I don't really like the jerry-ext/extfunc/ directory name. Common (jerry-ext/common-js-funcs) or non-standard JS function (jerry-ext/extra-js-funcs) would be a better name. Anything that represents its purpose (JS functions which are not part of the standard but generally useful).